examples/get_started/03_figure_element.py

```diff
@@ -0,0 +1,80 @@
+"""
+3. Figure elements
+==================
+The figure shows the naming of figure elements in PyGMT.
+
+- :meth:`pygmt.Figure()`: having a number of plotting methods. Every new
+  figure must start with the creation of a :meth:`pygmt.Figure()` instance
+- ``frame``: setting plot or map boundaries (**WSNE**, **wsne**, or
+  **lbtr**), adding annotations, ticks, gridlines (**afg**), axis labels
```

Comment:
Hm, I am not completely happy with the formulation of this bullet point. Actually, the user can pass a combination of the single letters of `WSENE`, `wsne`, and `lbtr`.